snuba/datasets/schemas/tables.py

```diff
@@ -63,6 +64,7 @@ def __init__(
         *,
         local_table_name: str,
         dist_table_name: str,
+        storage_set_key: StorageSetKey,
```

Comment:
Is this going to be temporary? To connect to the discussion we had two days ago about multiple schemas per storage (distributed vs single node). In my understanding, when we will have all the DDL including the table engine for distributed table in our code base, we would likely have at least two TableSchema per storage (that support a distributed deployment), one for the local table and one for the distributed one. 
If that is the case, then we would not need to pass the storage set key to each since the table name would be hardcoded in every TableSchema.

Is that right ?

Reply:
It's not designed to be temporary, I think it'll be here for some time. Yes if we decide to have a separate local and distributed schema it will be removed, but I don't think that decision has been made. When we do make that decision, I also don't think DDL will be the only or even the main motivation for doing so.